### ISSUE TITLE
[help wanted] RotateTransformation transparent (white) fields become black after saving file

### ISSUE THREAD

**Issue by JTOne123** [luberda-molinet/FFImageLoading#1013]
### Description

I have the method for saving file after transformations

```
        public Task<Stream> GetImageStreamAsync()
        {
            ImageService.Instance.InvalidateMemoryCache();

            var task = ImageService.Instance.LoadStream((cancelToken) => FileInfo.GetImageStreamAsync());

            var taskWithTransforms = task
                    .WithCache(FFImageLoading.Cache.CacheType.Disk)
                    .Transform(GetTransformations());

            var isJpeg = Path.GetExtension(FileName).ToLower() == ".jpg";

            if (isJpeg)
                return taskWithTransforms.AsJPGStreamAsync(100);
            else
                return taskWithTransforms.AsPNGStreamAsync();
        }
```

After saving "white fields" are becoming black. Why does it happen?

### Steps to Reproduce

1) Rotate an image to get white fields
2) Save with rotation via my method

### Expected Behavior

White or Transparent colors OR property to the color

### Actual Behavior

Black color instead of White or Transparent.

### Basic Information

- Version with issue: Lastest
- Platform: Android and UWP (iOS did not test)

**Comment by daniel-luberda** [comment]
Could you attach some files, so I could test it?

**Comment by JTOne123** [comment]
Yes, sure

Here they are
![droid](https://user-images.githubusercontent.com/3457140/39188538-b3d5ef36-47d8-11e8-8c9e-ea5a94dadee9.jpg)
![uwp](https://user-images.githubusercontent.com/3457140/39188545-b6c9c0b4-47d8-11e8-9109-121e4b52d4cd.jpg)

**Comment by daniel-luberda** [comment]
I need source images (before transformation) rather than output images.

**Comment by JTOne123** [comment]
![1596e6e907c2ab08c95cef6aca203aee](https://user-images.githubusercontent.com/3457140/39188999-d268ac80-47d9-11e8-979f-92b86f5195e0.jpg)


![media-share-0-02-05-edcdb528f5a81af832498715373f07ea95222ad5c79a8a112334fd796ef94d91-b7c811bb-4686-4ec9-8948-da0750fba5c6](https://user-images.githubusercontent.com/3457140/39188762-4386af1c-47d9-11e8-8933-f08fbb6042c4.jpg)

**Comment by JTOne123** [comment]
The original images

**Comment by JTOne123** [comment]
Hi Daniel, do you any updates on it?

**Comment by daniel-luberda** [comment]
Could you try newest prerelease nuget?

**Comment by JTOne123** [comment]
I have tested it with version 2.4.0.812-pre. Unfortunately, it does not work on UWP and Droid (I didn't test on ios) :(

Should I set a specific parameter for it?

**Comment by daniel-luberda** [comment]
Could you make a repo then?

**Comment by JTOne123** [comment]
Yep, here it is https://github.com/luberda-molinet/FFImageLoading/compare/master...JTOne123:master
Please, try it out on Android or UWP

Also, I've checked iOS everything works well for current stable build and master branch

**Comment by JTOne123** [comment]
ohhh, good idea, thank you)))
